app/views/download.scala.html

```diff
@@ -107,7 +108,22 @@ <h3 id="older-versions">Previous releases</h3>
       <table class="releases">
       @releases.previous.groupBy(_.version.slice(0, 3)).toSeq.sortBy(_._1).reverse.map { group =>
```

Comment:
I noticed the `version.slice(0, 3)` is technically broken too. It's really trying to get the first two version components, but it assumes version numbers are one digit. I think what we really want is something like `_.version.split('.').take(2).mkString(".")`.